src/main/java/de/hpi/bpt/chimera/jcore/executionbehaviors/scripttasks/context/ChimeraContext.java

```diff
@@ -0,0 +1,169 @@
+
+    /**
+     * Initialize the ChimeraContext
+     *
+     * @param activityInstance  The activity instance of the script task.
+     */
+    public ChimeraContext(ActivityInstance activityInstance) {
+        this.activityInstance = activityInstance;
+
+        updateDataObjects();
+    }
+
+    /**
+     * Updates the dataObjects object to get the newest values of the data objects
+     */
+    private void updateDataObjects() {
+        DbSelectedDataObjects dbDataObjectSelection = new DbSelectedDataObjects();
+        List<Integer> dataIds = dbDataObjectSelection.getDataObjectSelection(getScenarioInstanceId(), getActivityInstanceId());
+
+        DataManager dataManager = new DataManager(activityInstance.getScenarioInstance());
+        dataObjects.clear();
+        for(int id : dataIds) {
+            dataObjects.add(dataManager.getDataObjectForId(id).get());
+        }
+    }
+
+    /**
+     * Writes a message to the log file.
+     *
+     * @param message   The message to log.
+     */
+    @Override
+    public void log(String message) {
+        logger.info(message);
+    }
+
+    /**
+     * Writes an error message to the log file.
+     *
+     * @param message   The message to log.
+     */
+    @Override
+    public void logError(String message) {
+        logger.error(message);
+    }
+
+    /**
+     * Returns the value of a given input data object of the activity.
+     *
+     * @param dataObjectName    The name of the data object to get the value from.
+     * @param attributeName     The name of the attribute to get.
+     * @return an object with the value of the attribute.
+     */
+    public Object getAttribute(String dataObjectName, String attributeName) {
+        for(DataObject dataObject : dataObjects) {
+            if (dataObject.getName().equals(dataObjectName)) {
+                for (DataAttributeInstance dataAttributeInstance : dataObject.getDataAttributeInstances()) {
+                    if (dataAttributeInstance.getName().equals(attributeName)) {
+                        return dataAttributeInstance.getValue();
+                    }
+                }
+            }
+        }
+        return null;
+    }
+
+    /**
+     * Sets the value of a given input data object of the activity.
+     *
+     * @param dataObjectName    The name of the incoming data object.
+     * @param attributeName     The name of the attribute to change.
+     * @param value             The value to set to in the data object.
+     */
+    public void setAttribute(String dataObjectName, String attributeName, String value) {
+        for(DataObject dataObject : dataObjects) {
+            if(dataObject.getName().equals(dataObjectName)) {
+                for(DataAttributeInstance dataAttributeInstance : dataObject.getDataAttributeInstances()) {
+                    if(dataAttributeInstance.getName().equals(attributeName)) {
+                        Map<Integer, String> map = new HashMap<>();
```

Comment:
I'm sorry we are responding that late. For some reason I haven't subscribed for this project. Shall we fix this naming?